cross-compile-riscv.sh

```diff
@@ -0,0 +1,33 @@
+#/bin/bash
+
+# Cross-compile for RISC-V in an Ubuntu 22.04 container
+
+# To run mount a clean checkout of the code inside the container and run the script:
+#
+#   docker run --rm --mount type=bind,source=${PWD},target=/k3s ubuntu:22.04 /bin/bash -c /k3s/cross-compile-riscv.sh
+
+dpkg --add-architecture riscv64
+apt-get update
+apt-get install -y wget curl git gcc-riscv64-linux-gnu g++-riscv64-linux-gnu pkg-config libseccomp-dev:riscv64 make zstd
+
+HOST_ARCH=$(uname -m)
+if [ ${HOST_ARCH} = aarch64 ]; then
+    HOST_ARCH=arm64
+fi
+
+wget -P /tmp "https://dl.google.com/go/go1.20.5.linux-${HOST_ARCH}.tar.gz"
+(cd /; tar -C /usr/local -xzf "/tmp/go1.20.5.linux-${HOST_ARCH}.tar.gz")
+rm -rf /tmp/go1.20.5.linux-${HOST_ARCH}.tar.gz
```

Comment:
Would love to see these looked up from upstream, as we do at
https://github.com/k3s-io/k3s/blob/30f414e6145248d63c87f92eb7e966d784abd707/scripts/validate#L28-L29

Reply:
This is deprecated. I now use the existing build container, so no component versions are hard-coded.